packages/rum/src/types/sessionReplay.ts

```diff
@@ -33,6 +33,8 @@ export type CreationReason =
   | 'view_change'
   | 'before_unload'
   | 'visibility_hidden'
+  | 'page_frozen'
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it could be nice to add the schema update inside this PR as well